.circleci/config.yml

```diff
@@ -53,45 +66,76 @@ jobs:
           root: .
           paths:
             - .
-  build:
+  build-for-testing:
     macos:
       xcode: *xcode_version
     resource_class: *resource_class
     steps:
       - attach_workspace:
           at: .
-      - install-tools # chaching currently fails, needs some further tweaks
       - run:
-          name: Build
-          command: cd src/xcode && bundle exec fastlane build
+          name: Build for testing
+          command: |
+            cd src/xcode && \
+            xcodebuild \
+              -workspace "ENA.xcworkspace" \
+              -scheme ENA \
+              -destination "platform=iOS Simulator,OS=14.4,name=iPhone 11" \
+              -testPlan AllTests \
+              -only-test-configuration "DE" \
+              -derivedDataPath "./DerivedData" \
+              -jobs `sysctl -n hw.ncpu` \
+              -parallelizeTargets \
+              -quiet \
+              build-for-testing
```

Comment:
Not actively used but other parameters allow a test without build. This would then be needed.